ebpf_extensions/neteventebpfext/netevent_ebpf_ext_event.c

```diff
@@ -555,15 +555,21 @@ _ebpf_netevent_push_event(_In_ netevent_event_t* netevent_event)
         _event_buffer_size = payload_size;
     }
 
-    C_ASSERT(sizeof(netevent_event_notify_context.netevent_event_md.header) == sizeof(PKTMON_EVT_STREAM_PACKET_HEADER));
-    memcpy(
-        &netevent_event_notify_context.netevent_event_md.header,
-        packetHeader,
-        sizeof(netevent_event_notify_context.netevent_event_md.header));
+    _event_buffer_data_start = _event_buffer + sizeof(PKTMON_EVT_STREAM_PACKET_HEADER);
 
-    memcpy(_event_buffer, payload_start, payload_size);
-    netevent_event_notify_context.netevent_event_md.payload_start = _event_buffer;
-    netevent_event_notify_context.netevent_event_md.payload_end = _event_buffer + payload_size;
+    // C_ASSERT(sizeof(netevent_event_notify_context.netevent_event_md.header) ==
```

Comment:
nit - you have some commented out code - did you want to either put that in, or remove it before check in?